includes/reader-activation/class-reader-data.php

```diff
@@ -291,14 +291,15 @@ public static function setup_reader_activity() {
 		 * Article view activity.
 		 */
 		if ( is_singular( 'post' ) ) {
+			global $post;
 			$activity = [
 				'action' => 'article_view',
 				'data'   => [
 					'post_id'    => get_the_ID(),
 					'permalink'  => get_permalink(),
 					'categories' => wp_get_post_categories( get_the_ID(), [ 'fields' => 'ids' ] ),
 					'tags'       => wp_get_post_tags( get_the_ID(), [ 'fields' => 'ids' ] ),
-					'author'     => get_the_author(),
+					'author'     => $post->post_author,
```

Comment:
It depends on how we'd like to use this data. The ID is more reliable when mapping out authors programmatically, like for a "favorite authors" campaign's criteria. WDYT?

Reply:
yes, I think the ID is better if we are storing data. I was just checking because I was not sure you had a different intention, since you were using get_the_author before